github/repos_contents.go

```diff
@@ -317,5 +317,8 @@ func (s *RepositoriesService) GetArchiveLink(ctx context.Context, owner, repo st
 	}
 
 	parsedURL, err := url.Parse(resp.Header.Get("Location"))
-	return parsedURL, newResponse(resp), err
+	if err != nil {
+		return nil, newResponse(resp), err
+	}
+	return parsedURL, newResponse(resp), nil
```

Comment:
I know this is another thing that is inconsistent across this repo, but while I'm thinking about it, let's go ahead and add a blank line between the error handling and the final return, please.

```suggestion
	}

	return parsedURL, newResponse(resp), nil
```

Reply:
- [x] Done.